src/NuGet.Services.BasicSearch.Cloud/NuGet.Services.BasicSearch.Cloud.ccproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProductVersion>2.7</ProductVersion>
+    <ProductVersion>2.9</ProductVersion>
```

Comment:
Going to do the 2.9 update in all projects (gallery and this one) on friday and make sure build server supports it all.